hash.c

```diff
@@ -1748,7 +1748,7 @@ set_proc_default(VALUE hash, VALUE proc)
  *  call-seq:
  *     Hash.new                          -> new_hash
  *     Hash.new(default_value)           -> new_hash
- *     Hash.new {|hash, key| block }     -> new_hash
+ *     Hash.new { |hash, key| ... }      -> new_hash
```

Comment:
Even thought some users prefer to have white space here `{ |`, it's not the canonical form in Ruby.

```
koyoko% grep -R "{ |" | wc -l
4979
koyoko% grep -R "{|" | wc -l 
9234
```

I'd prefer we just use the most common form which is `{|`.

In addition, while I like the use of `...`, this may also get confusing because it's now valid syntax in Ruby:

```
def foo(...)
  bar(...)
end

def bar(x)
  puts x
end

foo(10) # => 10
```

Personally, I would also use `...` and maybe it's still okay here. The word `block` seems wrong too because we don't put a block in the block... 

Rather than `...` we could show a real example `hash[key] = default_value`?

Reply:
`{|` -- fine by me.
`...` -- I'm eager to be guided.  We have a decades-long history of showing schema, with the examples in the text below.
Can others in core weigh in?

Reply:
We can just do whatever is best for the user. I think I prefer real code rather than `...` at least in this case.

Reply:
I changed this back to 'block' (agrees with 35 others in this file, and hundreds more in other files).

I started writing programmer's reference docs in 1976.  I can't bring myself to put example code into a schema such as a call-seq.  (Anyone else can, though.)